kernelci/config/build.py

```diff
@@ -304,14 +331,19 @@ def __init__(self, name, tree, branch, variants):
 
         *variants* is a list of BuildVariant objects, to define all the
                    variants to build for this tree / branch combination.
+
+        *reference* is a Reference object which defines the tree and branch
```

Comment:
This might need a few more words, it defines the tree and branch to use as a reference in bisections when there is no base commit for the good and bad revisions.

Reply:
Corrected